lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -787,6 +794,42 @@ def _section_open_response_assessment(request, course, openassessment_blocks, ac
     return section_data
 
 
+def _section_recap(request, course, recap_blocks, access):
+    """Provide data for the Recap dashboard section."""
+
+    course_key = course.id
+    recap_block = recap_blocks[0]
+    block, __ = get_module_by_usage_id(
+        request, unicode(course_key), unicode(recap_block.location),
```

Comment:
Sorry, I missed this one, can you split all the parameters and add the keyword name? @andrey-canon